app/lib/doenet-data.ts

```diff
@@ -5,12 +5,17 @@ import { unstable_noStore as noStore } from "next/cache";
 const prisma = new PrismaClient();
 
 export async function createDocument(owner: number) {
+  let defaultDoenetmlVersion = await prisma.doenetmlVersions.findFirstOrThrow({
```

Comment:
This means we get an error trying to create a document if we haven't yet added a default row to `doenetmlVersions`. Is this a bad design? It only affects developers after they initialize the database, and I assume we could eventually seed the database for developers. I don't want to make doenetmlVersion optional.

Reply:
OK, added seeding via prisma